tools/azure-pipelines/build-apache-repo.yml

```diff
@@ -37,9 +37,12 @@ trigger:
 resources:
   containers:
   # Container with Maven 3.2.5, SSL to have the same environment everywhere.
-  # see https://github.com/flink-ci/flink-ci-docker
+  # see https://github.com/apache/flink-connector-shared-utils/tree/ci_utils
+  # The Docker image was customized for 1.17 to cover the changes of FLINK-18356
+  # (see chesnay/flink-ci:java_8_11_17_maven_386_v3 which is used in branches for
+  # Flink 1.18+) but still use Maven 3.2.5 (to avoid backporting FLINK-28016)
```

Comment:
should we refer to concrete image here?
I'm asking since now there is a new one available `chesnay/flink-ci:java_8_11_17_21_maven_386` which potentially could be used for 1.19